Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.cpp

```diff
@@ -475,6 +507,41 @@ int CLuaCameraDefs::SetCameraGoggleEffect(lua_State* luaVM)
     return 1;
 }
 
+bool CLuaCameraDefs::SetCameraUnderwaterEffectEnabled(bool bEnabled) noexcept
+{
+    g_pMultiplayer->SetUnderwaterEffectEnabled(bEnabled);
+
+    return true;
+}
+
+bool CLuaCameraDefs::SetCameraUnderwaterEffectSpeed(float fSpeed, std::optional<float> fFrequency)
```

Comment:
This is incompatible with the declaration given in the header: `static bool SetCameraUnderwaterEffectSpeed(float speed, float frequency);`